ui-particles-angular/projects/ui-particles-angular/src/scss/gio-mat-palettes.scss

```diff
@@ -35,13 +47,25 @@ $mat-primary-palette: mat.define-palette(
 // Accent
 $mat-accent-palette: mat.define-palette(
   (
-    darker: #504190,
```

Comment:
I hope all this colors change not break stuff in apim and cockpit. I haven't check 🤷‍♂️

Reply:
I will check

Reply:
<img width="1690" alt="Capture d’écran 2022-10-04 à 14 51 01" src="https://user-images.githubusercontent.com/1457497/193824891-99b8b945-a522-41b4-b6e7-334dca3f591c.png">
<img width="1668" alt="Capture d’écran 2022-10-04 à 14 50 45" src="https://user-images.githubusercontent.com/1457497/193824897-a6396509-8af4-4bb5-8d9d-85dbbc24630a.png">
<img width="224" alt="Capture d’écran 2022-10-04 à 14 49 27" src="https://user-images.githubusercontent.com/1457497/193824898-b38fe51a-c15e-4807-9f2d-46235c8de354.png">

Reply:
Lol, we have to change things in perseverance and adapt it to real life IMHO. 

Can I open a pr in APIM with this preview and ask validation by PO to all this change? 
Because for me is not valid (not your change but perseverance color !)